### PR TITLE
Handle COORDINATOR_LOAD_IN_PROGRESS in err_action

### PR DESCRIPTION
**Pull request by milindl**
See #4030 -- 

> It has been reported that rd_kafka_consumer_poll is returning an COORDINATOR_LOAD_IN_PROGRESS error when a cluster is rolled, evidently originating from two different protocol requests, as two different error strings are reported: Broker: Coordinator load in progress and JoinGroup failed: Broker: Coordinator load in progress.
> 
> At leas in the JoinGroup case, the java client backs off and retries: https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java#L639
> 
> This seems like an appropriate action for any protocol request to me, and propose adding this to rd_kafka_err_action

I don't have access to push to the relevant branch to update the same PR, so I've created this new PR which builds on top of it. This addresses the test addition comment, and the changelog comment, both.

### CODE REVIEW

File: tests/0117-mock_errors.c [hunk 1]
```diff
@@ -255,6 +255,57 @@ static void do_test_offset_commit_request_timed_out(rd_bool_t auto_commit) {
         SUB_TEST_PASS();
 }
 
+/**
+ * @brief PR #4030. A COORDINATOR_LOAD_IN_PROGRESS error is returned while a
```
Comment (edenhill) [inline tests/0117-mock_errors.c:259]: It is fine referencing an issue for background information, but we typically don't reference PRs, so just remove the 'PR #4030' part.

Comment (edenhill) [reply tests/0117-mock_errors.c:259]: It is not clear what "returned" refers to here, so flesh it out to say "returned from consumer_poll()" or similar.

**Review by edenhill — CHANGES_REQUESTED** [review]
Good stuff! Just comments on comments :)

File: tests/0117-mock_errors.c (hunk 1 above)
Comment (milindl) [reply tests/0117-mock_errors.c]: Done

File: tests/0117-mock_errors.c [hunk 2]
```diff
@@ -255,6 +255,58 @@ static void do_test_offset_commit_request_timed_out(rd_bool_t auto_commit) {
         SUB_TEST_PASS();
 }
 
+/**
+ * @brief A COORDINATOR_LOAD_IN_PROGRESS error is returned from
```
Comment (edenhill) [inline tests/0117-mock_errors.c:259]: Sorry but this might make sense now, but when reading this in the future it will seem as if we're expecting consumer_poll() to return COORDINATOR_LOAD_IN_PROGRESS. So maybe rephrase as:
"Verify that a cluster roll does not cause consumer_poll() to return the temporary and retriable COORDINATOR_LOAD_IN_PROGRESS error. We should backoff and retry in that case."

**Review by edenhill — CHANGES_REQUESTED** [review]

File: tests/0117-mock_errors.c (hunk 2 above)
Comment (milindl) [reply tests/0117-mock_errors.c]: Makes sense, especially since there's nothing to refer to besides this comment. I changed it to be like what you have recommended.

**Comment by edenhill** [comment]
Fix this build error:
https://app.travis-ci.com/github/edenhill/librdkafka/jobs/587553400#L708

**Review by edenhill — APPROVED** [review]
LGTM!
